kuma/core/utils.py

```diff
@@ -41,6 +41,13 @@ def to_html(pq):
     return pq.html(method='html')
 
 
+def is_beta(request):
```

Comment:
Nit: Someday "beta" will be the primary domain, and "wiki" will be the secondary. Start thinking about what these tests look like then. Do all the ``{% if beta %}`` stay and the ``{% else %}`` branches go away? Do they become ``{% if main %}`` or ``{% if published %}`` or something else? I think we should optimize for making that transition in as few lines of code as possible, and I'm worried about a lot of changes around ``beta``, or worse, a 2021 document saying "``beta`` refers to the version of the site after 2019..."